modules/fbs-core/api/src/main/kotlin/de/thm/ii/fbs/controller/v2/PlaygroundController.kt

```diff
@@ -0,0 +1,118 @@
+package de.thm.ii.fbs.controller.v2
+
+import com.fasterxml.jackson.databind.node.ArrayNode
+import de.thm.ii.fbs.model.v2.security.LegacyToken
+import de.thm.ii.fbs.model.v2.playground.*
+import de.thm.ii.fbs.model.v2.playground.api.SqlPlaygroundDatabaseCreation
+import de.thm.ii.fbs.model.v2.playground.api.SqlPlaygroundQueryCreation
+import de.thm.ii.fbs.model.v2.playground.api.SqlPlaygroundResult
+import de.thm.ii.fbs.services.v2.checker.SqlPlaygroundCheckerService
+import de.thm.ii.fbs.services.v2.persistence.*
+import de.thm.ii.fbs.utils.v2.annotations.CurrentToken
+import de.thm.ii.fbs.utils.v2.exceptions.NotFoundException
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@RequestMapping(path = ["/api/v2/playground/{uid}/databases"])
+class PlaygroundController(
+        private val userRepository: UserRepository,
+        private val databaseRepository: SqlPlaygroundDatabaseRepository,
+        private val entityRepository: SqlPlaygroundEntityRepository,
+        private val queryRepository: SqlPlaygroundQueryRepository,
+        private val sqlPlaygroundCheckerService: SqlPlaygroundCheckerService,
+) {
+    @GetMapping
+    @ResponseBody
+    fun index(@CurrentToken currentToken: LegacyToken): List<SqlPlaygroundDatabase> = databaseRepository.findByOwner_Id(currentToken.id)
+
+    @PostMapping
+    @ResponseBody
+    fun create(@CurrentToken currentToken: LegacyToken, @RequestBody database: SqlPlaygroundDatabaseCreation): SqlPlaygroundDatabase {
```

Comment:
while creating an database the last one is allways updated and no new one is created